tideways.c

```diff
@@ -1412,7 +1412,7 @@ long tw_trace_callback_curl_exec(char *symbol, zend_execute_data *data TSRMLS_DC
 	zval params[1];
 	ZVAL_RES(&params[0], Z_RES_P(argument));
 
-	if (SUCCESS == call_user_function_ex(EG(function_table), NULL, &fname, retval_ptr, 1, params, 1, NULL TSRMLS_CC)) {
+	if (SUCCESS == call_user_function_ex(EG(function_table), NULL, &fname, retval_ptr, 1, params, 1, NULL)) {
```

Comment:
TSRMLS_JUNK isn't necessary in PHP 7 specific code, I didn't have to remove that ... though you may want to adopt omitting it :)